### ISSUE TITLE
Notification badges in space panel are tiny

### ISSUE THREAD

**Issue by robintown** [element-hq/element-web#34648]
### Steps to reproduce

1. Get notifications in a space
2. Look at the space panel

#### What did you expect?

The number should have some padding for readability

#### What happened instead?

Tiny badge:

<img width="71" height="132" alt="Image" src="https://github.com/user-attachments/assets/3e2bd29e-df99-4167-9fda-261ebe747cec" />

### Operating system

NixOS 26.05

### Browser information

Firefox 153.0.3

### URL for webapp

develop.element.io

### Homeserver

Element version: ce74b5f614cb-js-178dc6528ebc Crypto version: Rust SDK 0.18.0 (16a138b), Vodozemac 0.10.0

### Will you send logs?

No

**Comment by ZacksBot** [comment]
This is a regression from https://github.com/element-hq/element-web/commit/ce74b5f614cbf27dff76c983f238d470d0a68346, likely due to the change from a `div` to a `button`. The fix is probably to remove the default button CSS applied by the browser

**Comment by daniellekirkwood** [comment]
+1 and the border on the pill looks too 'light' maybe this is why it looks squished... 

<img width="53" height="39" alt="Image" src="https://github.com/user-attachments/assets/b2907e4c-0c5f-47b1-acb6-9e83e077d6a1" />

<img width="63" height="45" alt="Image" src="https://github.com/user-attachments/assets/2ea446da-0e25-4bae-aee9-423ebd5640f2" />

When you get into triple digits it looks ok again, but the border still feels 'wrong'
<img width="59" height="44" alt="Image" src="https://github.com/user-attachments/assets/31639097-2ef4-4581-80eb-9c21ad8c1d66" />
